Artifacts/windows-iis/artifact.ps1

```diff
@@ -0,0 +1,34 @@
+$artifactLogFolder = Join-path -path $env:ProgramData -childPath 'DTLArt_IIS'
+New-Item -ItemType Directory -Force -Path $artifactLogFolder
+
+$winVerString = (Get-WmiObject Win32_OperatingSystem).Caption
+Write-Output "Detected Windows Version: $winVerString"
+$scriptFolder = Split-Path $Script:MyInvocation.MyCommand.Path
+
+if ($winVerString -like 'Microsoft Windows Server 2008 R2*' -or $winVerString -like 'Microsoft Windows Server 2012*'){
+    Write-Output 'Installing IIS on Windows Server 2008 or 2012'
+	$localLog = Join-Path $artifactLogFolder 'IISArtifact.log'
```

Comment:
Make sure the file is properly indented. Ideally, convert any Tab characters with Spaces.

Reply:
done